tests/integrated/test-region-iterator/runtest

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+#
+# Run the test, check it completed successfully
+#
+
+from __future__ import print_function
+try:
+  from builtins import str
+except:
+  pass
+from boututils.run_wrapper import shell, shell_safe, launch_safe, getmpirun
+from boutdata.collect import collect
+from sys import exit
+
+MPIRUN=getmpirun()
+
+print("Making Region Iterator test")
+s, out = shell_safe("make > make.log",pipe=True)
+if s:
+  print("Failed to run make")
+  exit(s);
+
```

Comment:
The `_safe` versions are throwing.
No need to test the output, `s` is only returned if it `0`.
See:
https://github.com/boutproject/BOUT-dev/blob/next/tools/pylib/boututils/run_wrapper.py#L196
Same goes for the calls below.

Reply:
Ah yes, thanks.